File: EDD_Proyecto1_Fase1/structs/Student.go
package structs

import (
	"strconv"
)

type Student struct {
	Id       int
	Name     string
	LastName string
	Password string
	Binnacle *Binnacle
}

func (student *Student) Json() string {

	return "{\"nombre\":\"" + student.Name + " " + student.LastName + "\",\"carnet\":" + strconv.Itoa(student.Id) + ",\"password\":\"" + student.Password + "\",\"Carpeta_Raiz\":\"/\" }"
}

type StudentNode struct {
	Next *StudentNode
	Prev *StudentNode
	Data Student
}

File: EDD_Proyecto1_Fase3/core/student.js

export class Student {

    constructor(name, id, password, rootFolder) {
        this.name = name;
        this.id = id;
        this.password = password;
        this.rootFolder = rootFolder;
    }

    getValue() {
        return this.id;
    }

    getGraphvizLabel() {
        return this.id + "\\n" + this.name;
    }


}